jenkins/jobs/cd/tiup_mirror_online_rc.groovy

```diff
@@ -0,0 +1,109 @@
+// not able to use job dsl, it has password params, must refract the pipeline script
+pipelineJob('tiup-mirror-online-ga') {
+    definition {
+        cpsScm {
+            lightweight(true)
+            scriptPath('jenkins/pipelines/cd/tiup/tiup-mirror-update-multi-products.groovy')
+            scm {
+                git{
+                    remote {
+                        url('https://github.com/PingCAP-QE/ci.git')
+                    }
+                    branch('main')
+                    extensions {
+                        cloneOptions {
+                            depth(1)
+                            shallow(true)
+                            timeout(5)
+                        }
+                    }
+                }
+            }
+        }
+    }
+    properties {
+        disableConcurrentBuilds{}
```

Comment:
`disableConcurrentBuilds()`?

Reply:
the sample use '{', it may contain other instruction in it